### PR TITLE
Performance audit tool

### PR DESCRIPTION
**Pull request by aidnurs**
## Basics

- [ ] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildserver is happy.
- [ ] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I fixed all affected decisions
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I mentioned [every code or binary](/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is conforming to [our Documentation Guidelines](/doc/documentation.md)
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to our Coding Guidelines
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)

### CODE REVIEW

**Review by badnames — APPROVED** [review]
Nice work. This should serve us well as a basis for further benchmarks!

File: benchmarks/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+npm run benchmark
+```
+
+## Benchmarking
+
+The benchmarking script runs a performance audit on a web page using `Lighthouse` and `Playwright`.
+The audit measures the performance of a web page by generating a performance score and then saves the results of the audit in `report` folder:
+
+-   `<test_name>-report.json` - the raw report of LightHouse for a single test case defined in `performance-audit.spec.js`
+-   `<timestamp>-lighthouse-results.csv` - the results of the audit for all test cases
```
Comment (markus2330) [inline benchmarks/README.md:40]: Why not have test_name here, too?

**Review by markus2330 — APPROVED** [review]
Great job, this will hopefully very useful for further benchmarks.

File: benchmarks/README.md (hunk 1 above)
Comment (aidnurs) [reply benchmarks/README.md:40]: the mentioned file is the result of an entire test run i.e. all test cases and not a single specific test case

File: benchmarks/playwright.config.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        //   name: 'Microsoft Edge',
+        //   use: { ...devices['Desktop Edge'], channel: 'msedge' },
+        // },
+        // {
+        //   name: 'Google Chrome',
+        //   use: { ..devices['Desktop Chrome'], channel: 'chrome' },
+        // },
+    ],
+
+    /* Run your local dev server before starting the tests */
```
[the remark this answers is not in the document]
Comment (aidnurs) [reply benchmarks/playwright.config.js:62]: Legit suggestion, I see this step in the CI pipeline for sure! during the local run, I'm not sure if the effort is worth implementing it

**Comment by markus2330** [comment]
Thx, great job!

**Pull request merged**